### PR TITLE
Issue #320 fixed , Date format Follows system defaults

### PR DESCRIPTION
**Pull request by lokeshkvn**

### CODE REVIEW

**Comment by lokeshkvn** [comment]
* Date follows system defaults 
* Removed Redundant code 
* removed unused namesapce declaration 
```
xmlns:tools="http://schemas.android.com/tools"
```

File: collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesActivity.java [hunk 1]
```diff
@@ -44,10 +44,34 @@
 import org.odk.collect.android.logic.FormController;
 import org.odk.collect.android.logic.PropertyManager;
 import org.odk.collect.android.utilities.MediaUtils;
-import static org.odk.collect.android.preferences.PreferenceKeys.*;
 
 import java.util.ArrayList;
 
+import static org.odk.collect.android.preferences.PreferenceKeys.ARRAY_INDEX_GOOGLE_MAPS;
```
Comment (dcbriccetti) [inline collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesActivity.java]: Would it be reasonable to import PreferenceKeys.* here?

**Comment by lognaturel** [comment]
Welcome @lokeshkvn and thanks for looking into this! We try hard to keep pull requests very focused and only affecting a couple of files so they are easier to review and less likely to introduce unrelated issues. If you would like to be involved in some reformatting and refactoring of the code, please look at some of the issues labeled 'refactor'. Please resubmit only the changes related to the date fix.

**Comment by lokeshkvn** [comment]
Okay

**Pull request closed without merging**